### ISSUE TITLE
Annoying error message &  Pods not running

### ISSUE THREAD

**Issue by stevef1uk** [code-ready/crc#663]
I am seeing a lot of the following errors. Sometimes, if I leave things they resolve themselves, most times they don't:

'0/1 nodes are available: 1 node(s) had taints that the pod didn't tolerate.'

I am seeing these a lot on my MacBook & Wondows Laptop (much worse as less memory can be given to the VM) along with low ephemeral storage warnings. Pre 4.1 I would have used docker to purge exited container images to free up disk space, not sure how one does this with v4.1 when there is no Docker?

I am still seeing a lot of these taint errors on my iMac though where I have given the VM 28GB and I have a 2TB of disk.

Basically, most times I try to install more complex  applications such as Code Ready Operators, or OpenShift Service Mesh I am getting these errors and thus can't run them.

Help appreciated.

Regards

Steve

**Comment by anpati1** [comment]
On Windows CRC. Some pods are not being scheduled and remain in pending status due to
`Warning  FailedScheduling  5s (x2 over 75s)  default-scheduler  0/1 nodes are available: 1 node(s) had taints that the pod didn't tolerate`
oc describe for the pod
`PS C:\crc> oc describe pod -n istio-system istio-egressgateway-757d97cc65-r2vdd
Name:               istio-egressgateway-757d97cc65-r2vdd
Namespace:          istio-system
Priority:           0
PriorityClassName:  <none>
Node:               <none>
Labels:             app=istio-egressgateway
                    chart=gateways
                    heritage=Tiller
                    istio=egressgateway
                    maistra-control-plane=istio-system
                    maistra-version=1.0.0
                    pod-template-hash=757d97cc65
                    release=istio
Annotations:        openshift.io/scc: restricted
                    sidecar.istio.io/inject: false
Status:             Pending
IP:
Controlled By:      ReplicaSet/istio-egressgateway-757d97cc65
Containers:
  istio-proxy:
    Image:       registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:1.0.0
    Ports:       8080/TCP, 8443/TCP, 15443/TCP, 15090/TCP
    Host Ports:  0/TCP, 0/TCP, 0/TCP, 0/TCP
    Args:
      proxy
      router
      --domain
      $(POD_NAMESPACE).svc.cluster.local
      --log_output_level=default:info
      --drainDuration
      45s
      --parentShutdownDuration
      1m0s
      --connectTimeout
      10s
      --serviceCluster
      istio-egressgateway
      --zipkinAddress
      zipkin:9411
      --proxyAdminPort
      15000
      --statusPort
      15020
      --controlPlaneAuthPolicy
      NONE
      --discoveryAddress
      istio-pilot:15010
    Limits:
      cpu:     2
      memory:  256Mi
    Requests:
      cpu:      100m
      memory:   128Mi
    Readiness:  http-get http://:15020/healthz/ready delay=1s timeout=1s period=2s #success=1 #failure=30
    Environment:
      POD_NAME:                     istio-egressgateway-757d97cc65-r2vdd (v1:metadata.name)
      POD_NAMESPACE:                istio-system (v1:metadata.namespace)
      INSTANCE_IP:                   (v1:status.podIP)
      HOST_IP:                       (v1:status.hostIP)
      ISTIO_META_POD_NAME:          istio-egressgateway-757d97cc65-r2vdd (v1:metadata.name)
      ISTIO_META_CONFIG_NAMESPACE:  istio-system (v1:metadata.namespace)
      ISTIO_META_ROUTER_MODE:       sni-dnat
    Mounts:
      /etc/certs from istio-certs (ro)
      /etc/istio/egressgateway-ca-certs from egressgateway-ca-certs (ro)
      /etc/istio/egressgateway-certs from egressgateway-certs (ro)
      /var/run/secrets/kubernetes.io/serviceaccount from istio-egressgateway-service-account-token-j8s8l (ro)
Conditions:
  Type           Status
  PodScheduled   False
Volumes:
  istio-certs:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  istio.istio-egressgateway-service-account
    Optional:    true
  egressgateway-certs:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  istio-egressgateway-certs
    Optional:    true
  egressgateway-ca-certs:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  istio-egressgateway-ca-certs
    Optional:    true
  istio-egressgateway-service-account-token-j8s8l:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  istio-egressgateway-service-account-token-j8s8l
    Optional:    false
QoS Class:       Burstable
Node-Selectors:  <none>
Tolerations:     node.kubernetes.io/memory-pressure:NoSchedule
                 node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason            Age               From               Message
  ----     ------            ----              ----               -------
  Warning  FailedScheduling  5s (x2 over 75s)  default-scheduler  0/1 nodes are available: 1 node(s) had taints that the pod didn't tolerate.`

**Comment by gbraad** [comment]
I am not sure what to say about this seems to be related to istio and how resources gets assigned. Since no follow-up or feedback, closing seems aacceptable.
